### PR TITLE
Inital commit of ipmi driver. DON'T MERGE !

### PR DESCRIPTION
**Pull request by sergeygalkin**
For informational purposes

### CODE REVIEW

File: devops/driver/ipmi/ipmi_driver.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+                     "prompt 0\n"
+                     "MENU TITLE Fuel Installer\n"
+                     "TIMEOUT 20\n"
+                     "LABEL LABEL fuel\n"
+                     "MENU LABEL Install ^FUEL\n"
+                     "MENU DEFAULT\n"
+                     "KERNEL /fuel/isolinux/vmlinuz\n"
+                     "INITRD /fuel/isolinux/initrd.img\n"
+                     "APPEND ks=nfs:%s:%s/tftpboot/fuel/ks.cfg "
+                     "ks.device='eth0' "
```
Comment (adidenko) [inline devops/driver/ipmi/ipmi_driver.py:152]: On some bare-metal environments we may need to customize ks.device interface.

File: devops/driver/ipmi/ipmi_driver.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+        menu_boot = ("DEFAULT menu.c32\n"
+                     "prompt 0\n"
+                     "MENU TITLE Fuel Installer\n"
+                     "TIMEOUT 20\n"
+                     "LABEL LABEL fuel\n"
+                     "MENU LABEL Install ^FUEL\n"
+                     "MENU DEFAULT\n"
+                     "KERNEL /fuel/isolinux/vmlinuz\n"
+                     "INITRD /fuel/isolinux/initrd.img\n"
+                     "APPEND ks=nfs:%s:%s/tftpboot/fuel/ks.cfg "
```
Comment (adidenko) [inline devops/driver/ipmi/ipmi_driver.py:151]: On bare-metal environments we may need to use custom kick-start config.

File: devops/driver/ipmi/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+__author__ = 'sgalkin'
```
Comment (xarses) [inline devops/driver/ipmi/__init__.py:1]: **init**.py should be empty

File: devops/driver/ipmi/ipmi_driver.py [hunk 4]
```diff
@@ -0,0 +1,249 @@
+# vim: ts=4 sw=4 expandta
```
Comment (xarses) [inline devops/driver/ipmi/ipmi_driver.py:1]: Add License Stub. You can put author here instead of **init**.py, it should be Full name and <handle | email>

**Pull request closed without merging**